brevproduksjon/src/main/java/no/nav/foreldrepenger/fpformidling/brevproduksjon/bestiller/DokgenBrevproduksjonTjeneste.java

```diff
@@ -74,9 +74,10 @@ public DokgenBrevproduksjonTjeneste(DokumentFellesDataMapper dokumentFellesDataM
     public void bestillBrev(DokumentHendelse dokumentHendelse, Behandling behandling, DokumentMalType dokumentMal) {
         var bestill = BestillingType.BESTILL;
         DokumentData dokumentData = lagreDokumentDataFor(behandling, dokumentMal, bestill);
-
+        int teller = 0;
         for (DokumentFelles dokumentFelles : dokumentData.getDokumentFelles()) {
             byte[] brev = genererDokument(dokumentHendelse, behandling, dokumentMal, dokumentFelles, bestill);
+            var unikBestillingsUuidPerDokFelles = dokumentHendelse.getBestillingUuid() + "-" + teller++;
```

Comment:
Jeg er litt glad i at vi bruker referanse konsekvent der mulig men legger på suffix for kopiene
Noko slikt
var unikBestillingsUuidPerDokFelles = dokumentHendelse.getBestillingUuid();
if (teller > 0) {
   unikBestillingsUuidPerDokFelles = unikBestillingsUuidPerDokFelles + "-" + teller;
 }
teller++;

Reply:
Enig - da har vi det noe ryddigere i bestillingene hvor det kun er en mottaker (flesteparten).

Reply:
Ja, enig